src/LineHandler.mss

```diff
@@ -0,0 +1,242 @@
+        // TODO: The Nebenstimme line type only has a start symbol and an end symbol without an actual line.
+        'line.staff.nebenstimme',              CreateSparseArray('Line', CreateDictionary('startsym', 'N', 'endsym', 'angledown', 'lwidth', '0')),
+        // 'line.staff.pick.scrape',              CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.rake',                     CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.slide',                    CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.1',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.2',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.3',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.4',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.5',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.6',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.7',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.above.8',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.1',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.2',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.3',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.4',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.5',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.6',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.7',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        // 'line.staff.string.below.8',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
+        'line.staff.vertical',                 CreateSparseArray('Line', CreateDictionary('form', 'solid', 'type', verticalLine)),
+
+        // Type = 'OctavaLine'
+        'line.staff.octava.minus15', CreateSparseArray('Octave', CreateDictionary('dis', '15', 'place', 'below')),
+        'line.staff.octava.minus8', CreateSparseArray('Octave', CreateDictionary('dis', '8', 'place', 'below')),
+        'line.staff.octava.plus15', CreateSparseArray('Octave', CreateDictionary('dis', '15', 'place', 'above')),
```

Comment:
```suggestion
        'line.staff.octava.plus15', CreateSparseArray('Octave', CreateDictionary('dis', '15', 'dis.place', 'above')),
```

Reply:
Thanks, @rettinghaus, for having such a careful look!